python-connectors/github_pull-requests/connector.json

```diff
@@ -0,0 +1,43 @@
+{
+    "meta" : {
+        "label": "GitHub Pull Requests",
+        "description": "Allows to list all Pull Request on the specified repositories",
+        "icon": "icon-refresh"
+    },
+    "readable": true,
+    "writable": false,
+
+    "params": [
+        {
+            "name": "personal_access_token_credentials_preset",
+            "label": "Github Personal Access Token",
+            "description": "For more information on how to create your token: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token",
+            "type":"PRESET",
+            "parameterSetId": "personal-access-token"
+        },
+        {
+            "name": "repositories",
+            "label": "Git full name repositories",
+            "description": "Repositories to fetch Pull Requests from (full name means in the form of {owner}/{repo})",
+            "type":"STRINGS"
+        },
+		{
```

Comment:
Out of the box from PyCharm those files are properly indented. I can reindent all to make sure there are no more bad indentation but it will change the default indendation of the file so not sure it's better 🤷‍♂️